manager/install.sh

```diff
@@ -368,6 +369,15 @@ function remove_nodegroups() {
   echo
 }
 
+func setup_ipvs() {
+  kube_proxy_pod=$(kubectl get pod -n kube-system -l k8s-app=kube-proxy -o jsonpath='{.items[*].metadata.name}' | cut -d " " -f1)
+  kubectl exec -it -n kube-system ${kube_proxy_pod} -- cat /var/lib/kube-proxy-config/config > proxy_config.yaml
+  python upgrade_kube_proxy_mode.py proxy_config.yaml > proxy_config.yaml
```

Comment:
this sequence of operations seems a bit convoluted.

what is the goal here? can't you just patch the configmap directly?

Reply:
Yep, made it less convoluted. We need to add a new key in the data section of the config map.